src/OSPSuite.Presentation/Presenters/AbstractExplorerPresenter.cs

```diff
@@ -308,6 +325,8 @@ public virtual void RemoveChildrenClassifications(ITreeNode<IClassification> par
 
       public virtual ITreeNode AddClassifiableToTree<T>(T classifiable, RootNodeType rootNodeType, Func<ITreeNode<IClassification>, T, ITreeNode> addClassifiableToTree) where T : IClassifiable
       {
+         addClassifiableUnderActiveClassification(classifiable);
```

Comment:
this is the lowest method. We have a classifiable to add to the tree. We move it to the active classification if required